### PR TITLE
Changes required for script to run on Python 3.8.1

### PR DESCRIPTION
**Pull request by blackstarxr253**

### CODE REVIEW

**Comment by blackstarxr253** [comment]
The changes were made for the script to run on Windows 10 with Python 3.8

-Somnus

**Comment by blackstarxr253** [comment]
Also corrects Issue #1

**Comment by jilleb** [comment]
Great work! 
I'll test and merge asap.
Thanks!

File: odis2vcp.py [hunk 1]
```diff
@@ -160,6 +160,6 @@ def parse_small_oe_file():
 print ""
```
Comment (upgreydd) [inline odis2vcp.py:160]: This one not converted will make an issue
should be
```
print("")
```

**Review by upgreydd — CHANGES_REQUESTED** [review]

**Pull request closed without merging**